packages/main/src/plugin/proxy.spec.ts

```
@@ -77,67 +71,69 @@ async function buildProxy(): Promise<ProxyServer> {
-  );
-  const proxy = new Proxy(configurationRegistry, certificates);
-  await proxy.init();
+  await proxy?.setState(ProxyState.PROXY_MANUAL);
+  await proxy?.setProxy({
+    httpsProxy: `127.0.0.1:${address.port}`,
+    httpProxy: undefined,
+    noProxy: undefined,
+  });
+
   let connectDone = false;
   proxyServer.on('connect', () => (connectDone = true));
```

Biome (2.1.2)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)